.github/workflows/smoke.yml

```diff
@@ -624,8 +641,26 @@ jobs:
           cache-on-failure: true
       - name: Install cargo-llvm-cov
         uses: taiki-e/install-action@cargo-llvm-cov
+      - name: Fscache Setup
```

Comment:
Can we put these setup codes into a script file, so that we can reuse it between jobs.